frontend/src/components/ui/AppInput.vue

```diff
@@ -3,20 +3,23 @@ import AppInputError from './AppInputError.vue'
 </script>
 
 <template>
-
   <div class="text_label">
-    <label>{{label}}</label>
+    <label>{{ label }}</label>
+    <AppTooltip v-if="$slots.tooltip">
+      <slot name="tooltip"></slot>
```

Comment:
Are we using named slots here just in case we have a requirement for multiple slots in the future? Or because it makes the conditional rendering a little easier? (Or both)
Same comment applies to all the components that have a slot for a tool tip

Reply:
Exactly as you stated. The rationale is that the tooltip is not what you'd consider the main content so I made it a named slot to be clear about that. If we have multiple slots in the future (e.g. for a more complicated component) then this is a good approach. It also helps with the conditional rendering.